comp/otelcol/otlp/components/exporter/logsagentexporter/logs_exporter.go

```diff
@@ -64,7 +64,7 @@ func (e *Exporter) ConsumeLogs(ctx context.Context, ld plog.Logs) (err error) {
 		}
 	}()
 
-	payloads := e.translator.MapLogs(ctx, ld)
+	payloads := e.translator.MapLogs(ctx, ld, nil)
```

Comment:
Is this expected to be nil?

Reply:
Note this code path is also used in OSS collector

Reply:
Yes this is expected. I have opened this PR to quickly have a version of datadog-agent compatible with opentelemetry-mapping-go. I'll open a follow up PR to handle gateway usage in the Agent.

Reply:
Thx that makes sense to me